### PR TITLE
FOUR-11416 Two Factor Authentication General Settings

### PR DESCRIPTION
**Pull request by caleeli**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-11416

## Code Review Checklist
- [x] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [x] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [x] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [x] This solution fixes the bug reported in the original ticket.
- [x] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [x] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [x] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [x] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [x] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy
ci:APP_DEBUG=1
ci:TWILIO_ACCOUNT_SID="ACf3487ba53a2b560c6a50625068fd7bc3"
ci:TWILIO_AUTH_TOKEN="15a5b1955d7ff6cc394caaa1f8b70b41"
ci:TWILIO_ACTIVE_PHONE_NUMBER="+12057511892"

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-97443c91eb.eng.processmaker.net

File: ProcessMaker/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -231,6 +231,9 @@ public function beforeLogout(Request $request)
     {
         if (Auth::check()) {
             event(new Logout(Auth::user()));
+
+            // Always destroy 2fa flag
+            session()->remove('2fa-validated');
```
Comment (caleeli) [inline ProcessMaker/Http/Controllers/Auth/LoginController.php:236]: Consider to add this string into a constant:
- '2fa-validated'

**Review by caleeli — COMMENTED** [review]
Review the suggestion

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-97443c91eb.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-97443c91eb.eng.processmaker.net

**Pull request merged**